### DISCUSSION TITLE
authorization for normal users not superuser

### DISCUSSION THREAD

**Discussion by tirelibirefe** [strimzi/strimzi-kafka-operator#6202] [Q&A]
Hello,
I see `superUser `authorization in documentation.  That's ok.

What about about normal users? Are all users "superusers" but are some of them restricted via KafkaUser crd?

Thanks & Regards

**Comment by scholzj** [comment]
Please check the docs. They describe how to designate some users as super users, but also how to set specific ACLs using the KafkaUser resources.
